app/controllers/works_controller.rb

```diff
@@ -13,11 +13,15 @@ def index
   end
 
   # Renders the "step 0" information page before creating a new dataset
-  def new_submission; end
-
   def new
+    render "new_submission"
+  end
+
+  # Creates the new dataset
+  def new_submission
     default_collection_id = current_user.default_collection.id
-    work = Work.create_dataset("New Dataset", current_user.id, default_collection_id)
+    datacite_resource = datacite_resource_from_form
+    work = Work.create_dataset(datacite_resource.main_title, current_user.id, default_collection_id, datacite_resource)
     redirect_to edit_work_path(work, wizard: true)
   end
```

Comment:
Swapped the meaning of the `new` and `new_submission` methods and their respectively routes in `config.rb` and I think this makes more sense and it's more Rails-y. `New` is now a GET operation and `new_submission` is the POST.